lms/envs/aws.py

```diff
@@ -1099,3 +1099,8 @@
 
 # Allow extra middleware classes to be added to the app through configuration.
 MIDDLEWARE_CLASSES.extend(ENV_TOKENS.get('EXTRA_MIDDLEWARE_CLASSES', []))
+
+AUTHORIZATION_URL = ENV_TOKENS.get('HARAMBEE_AUTHORIZATION_URL', 'https://mvpdev.harambeecloud.com/identityserver/connect/authorize')
```

Comment:
These look like settings for the dev environment  to replace the hardcoded values for    'https://prod.harambeecloud.com/identityserver/connect/****** which are directed at the prod env

Reply:
this URLs will be configured via server-vars.yml into OpenEdx's lms.env.json conf file during deploy procedure. hardcoded UAT URLs left here intentionally to allow merging harambee/uat and harambee/production branches.